app/models/review_response_map.rb

```diff
@@ -182,4 +182,13 @@ def self.final_versions_from_reviewer(reviewer_id)
     end
     review_final_versions
   end
+
+  def self_review_response_map?
```

Comment:
We are not using that method. Wrote it when we started project but forgot to remove it. I will take care of this method.